new-frontiers-14/frontier-station-14
# Sprinter atmos rework

## About the PR
Added fore and aft air alarm.
Added vent and scrubber to bar.
Linked atmos devices to air alarms.

Reevaluated shuttle price.

Changed medbay airlock to be appropriately medical.
Replaced sodium lights with fluorescent lights.

## Why / Balance
Adding the air alarms and linking the devices brings the shuttle in line with current guidelines.
The shuttle's value has fallen since its last appraisal. Adding a 30% markup brings it in line with other expedition shuttles.
The medbay airlock should be properly marked.
This is a shuttle with a central bar. People are intended to be able to relax here, even if it is a bit grungy. Sodium lights run counter to this and are, frankly, annoying as a player to have ship-wide.

## How to test
Purchase the Sprinter. Walk around. Play with the atmos alarms to check they function (fore alarm in engineering, aft alarm in the main hallway). Check that the vents and scrubbers all have expected gas levels after a short while to ensure they are properly connected to the distribution and waste networks.

## Media
### Air Alarms
![airalarmaft](https://github.com/user-attachments/assets/a70eb8f8-52bb-43d7-a1e6-b32c2ceb0138)
![airalarmfore](https://github.com/user-attachments/assets/2ef7831d-c9e5-408d-8390-bf1d4f70344f)

### Medbay
![medbay](https://github.com/user-attachments/assets/9de61d70-2a69-4c4d-b08c-6d034a57c5f6)

## Requirements
